beesdoo_shift_attendance/security/ir.model.access.csv

```diff
@@ -6,7 +6,7 @@ create_beesdoo_shift_sheet_expected,create_beesdoo_shift_sheet_expected,model_be
 manage_beesdoo_shift_sheet_added,manage_beesdoo_shift_sheet_added,model_beesdoo_shift_sheet_added,group_shift_attendance_sheet,1,1,1,1
 create_beesdoo_shift_sheet,create_beesdoo_shift_sheet,model_beesdoo_shift_sheet,group_shift_attendance_sheet,1,1,1,0
 sheet_access_beesdoo_shift_template,sheet_access_beesdoo_shift_template,beesdoo_shift.model_beesdoo_shift_template,group_shift_attendance_sheet,1,0,0,0
-sheet_access_beesdoo_shift_type,sheet_access_beesdoo_shift_type,beesdoo_shift.model_beesdoo_shift_type,group_shift_attendance_sheet,1,0,0,0
+sheet_access_beesdoo_shift_type,sheet_access_beesdoo_shift_type,beesdoo_shift.model_beesdoo_shift_type,,1,0,0,0
```

Comment:
I don't know if it's recommended... no way to give it the the group portal ? As I suppose that you're trying to give the right access to the workers

Reply:
I've tried the following : adding `base.group_portal` as `group_id`. 
Unfortunately, it doesn't solve the issue: 
```
Désolé, vous n'êtes pas autorisé à accéder à ce document. Seuls les utilisateurs avec les niveaux d'accès suivants sont actuellement autorisés à faire cela:
- Cooperative management/Shift and Worker Read Access
- Cooperative management/Shifts and Attendance Sheets Management
- User types/Portal
- Cooperative management/Planning Management

(Modèle de document: beesdoo.shift.type) - (Opération: read, Utilisateur : 3390) 
```